src/Microsoft.Health.Fhir.Shared.Core/Features/Operations/Everything/PatientEverythingService.cs

```diff
@@ -299,16 +414,25 @@ public class PatientEverythingService : IPatientEverythingService
                 searchParameters.Add(Tuple.Create(KnownQueryParameterNames.ContinuationToken, continuationToken));
             }
 
+            // If we are processing a "seealso" link, the parent patient that links to it will be in its patient compartment
+            if (token.IsProcessingSeeAlsoLink)
+            {
+                // Add a parameter to prevent this, since we already returned the parent patient resource in a previous call
+                searchParameters.Add(Tuple.Create(KnownQueryParameterNames.Id, $"ne{parentPatientId}"));
```

Comment:
Does this prefix `ne` work for string? If the id is "abc", will it search for an id equal to `neabc`?

Reply:
Ah yes! This should be using a modifier instead of a prefix. Fixed in f8bd4af.